app/src/main/java/ru/appintheair/flightreviewssubmit/SubmitScreen.kt

```diff
@@ -0,0 +1,167 @@
+package ru.appintheair.flightreviewssubmit
+
+import android.graphics.drawable.Drawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
```

Comment:
This overscroll effect in the middle of the screen looks really strange
[video.mp4.zip](https://github.com/Pank-Off/FlightsApp/files/5537969/video.mp4.zip)

Reply:
I can hide the button and instead show the progress bar in place of the button. Its a normal solution?

Reply:
Yep, it'll do